dashboard/models.py

```diff
@@ -283,8 +283,8 @@ class Meta:
 
 class FileAccess(models.Model):
     id = models.AutoField(primary_key=True, verbose_name="Table Id")
-    file_id = models.CharField(blank=True, max_length=255, null=False, verbose_name='File Id')
-    user_id = models.CharField(blank=True, max_length=255, null=False, verbose_name='User Id')
+    file_id = models.BigIntegerField(blank=True, null=False, verbose_name='File Id')
+    user_id = models.BigIntegerField(blank=True, null=False, verbose_name='User Id')
```

Comment:
Due to this change, the file view is broken. It a minor change something like this needs to be added in the `views.py` 
```
df['file_id_name'] = df['file_id'].astype(str).str.cat(df['file_name'], sep=';')
```